freqtrade/rpc/rpc.py

```diff
@@ -11,7 +11,7 @@
 import psutil
 from dateutil.relativedelta import relativedelta
 from dateutil.tz import tzlocal
-from numpy import NAN, inf, int64, mean
+from numpy import nan as NAN, inf, int64, mean
```

Comment:
To start with - we'd surely not alias the import, but instead replace this throughout the file.

It's however also not really a problem at this point assuming you're using the correct (pinned) version of numpy.

While we'll be needing changes to support numpy 2.0 - i doubt this is the only one (and it'll surely not be done by alias-importing something) - so this PR as is unfortunately provides no value.


As is, this PR provides no benefit, so i'll be closing it.